plugins/atomicfu/atomicfu-runtime/build.gradle.kts

```diff
@@ -2,6 +2,7 @@ description = "Runtime library for the Atomicfu compiler plugin"
 
 plugins {
     kotlin("js")
+    `maven-publish`
 }
 
 group = "org.jetbrains.kotlin"
```

Comment:
I would propose to change name as well - could be done in `settings.gradle` after line 800:
```groovy
project(':kotlinx-atomicfu-runtime').name = 'atomicfu-runtime'
```

Reply:
Also note that this will require to change dependency name in other Kotlin projects